Bottom: b21774fffc2a8b4434a20a1a0aa743335f58ca4b
Top:    57d2df0fc68bc1cc8686912a29a17cf6b1bc6881
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-01 11:56:45 +0100

action.c: Fix the SetValue(TextScale, ...) action after nanometer conversion


---

diff --git a/src/action.c b/src/action.c
index efb5aab..14f23de 100644
--- a/src/action.c
+++ b/src/action.c
@@ -2217,7 +2217,7 @@ ActionSetValue (int argc, char **argv, Coord x, Coord y)
 
 	case F_Text:
 	case F_TextScale:
-	  value /= 45;
+	  value /= FONT_CAPHEIGHT;
 	  SetTextScale (absolute ? value : value + Settings.TextScale);
 	  break;
 	default:
